doc/cookbook/source/examples/evaluation/cross_validation_linear_svm.rst

```diff
@@ -0,0 +1,62 @@
+============================================
+Cross Validation on a Linear SVM Model
+============================================
+
+Cross Validation is a type of model validation technique for figuring out how the
+results from a statistical analysis will generalize to an independent data set. It
+essentially involves training the model on a subset of the data, and testing on the
+remainder, then repeating this process for several complementary partitions and then
+combining all the validation results generated (For instance, an average) to estimate
+a performance for the final predictive model.
+
+One type of cross validation is known as :math:`k-` cross validation.
+For instance, Suppose you have the training data :math:`\mathbf{x}_1,\mathbf{x}_2, ..., \mathbf{x}_{10n}`
+for a positive integer :math:`n` , then :math:`10-` cross validation might for instance partition the data into
+:math:`P_1, P_2, ..., P_n` each with :math:`10` data points. Then, the model would be trained on
+:math:`P_1, P_2, ...,P_{n-1}` and test it on :math:`P_n` to get accuracy :math:`a_1` . Then the model
+would train again from scratch on :math:`P_1, P_2, ...,P_{n-2}, P_{n}` and is tested on :math:`P_{n-1}` to get
+:math:`a_2`.  This process is repeated until we have :math:`a_1, a_2, ..., a_n`, at which we take their average,
```

Comment:
"This process is repeated  ... , of which we compute the average and standard deviation."

Remove: and assign that as the accuracy of this mode -- not clear what it means

Reply:
Done